myst_nb/_static/mystnb.js

```diff
@@ -1,44 +0,0 @@
-// Initialize MathJax with the notebook config
```

Comment:
@choldgraf is there anything here you could see as being an issue because we've removed it

Reply:
Hmmm I don't _think_ so as long as we have the same behavior for catching math syntax delimeters. The notebook mathjax config is actually pretty non-specific and isn't strongly documented anywhere, so we have a little bit of wiggle room in general, we just don't want to surprise people with MyST's behavior I think. Let's try it out and see if people start complaining :-)